drivers/ieee802154/ieee802154_rf2xx.c

```diff
@@ -378,6 +382,45 @@ static int rf2xx_filter(struct device *dev,
 	return -ENOTSUP;
 }
 
+#if !defined(CONFIG_IEEE802154_RAW_MODE) && \
+	defined(CONFIG_NET_L2_OPENTHREAD)
+
+#define ACK_FRAME_LEN 3
+#define ACK_FRAME_TYPE (2 << 0)
+#define ACK_FRAME_PENDING_BIT (1 << 4)
+
+static void rf2xx_handle_ack(struct rf2xx_context *ctx, u8_t seq_number)
+{
+	struct net_pkt *pkt;
+	u8_t ack_psdu[ACK_FRAME_LEN] = {ACK_FRAME_TYPE, 0, seq_number};
```

Comment:
reverse the order of these 2 lines, we usually follow "pre-assigned variables first, then the rest" in net stack. (I don't think it's documented anywhere though)